util/tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+    "__comment": "setting for running gen-typescript-decl.ts",
+    "compilerOptions": {
+        "target": "es6",
+        "noImplicitAny": true,
+        "sourceMap": false,
+        "moduleResolution": "node",
+        "listEmittedFiles": false,
+        "emitDecoratorMetadata": true,
+        "pretty": true,
+        "module": "commonjs",
+        "importHelpers": true
+    },
+    "compileOnSave": false,
+    "atom": {
+        "rewriteTsconfig": false
+    }
+}
```

Comment:
It seems we can remove that file and still run the `.ts` code. Guess I should remove it.